policyengine_us/parameters/gov/hhs/chip/spending/total/total.yaml

```diff
@@ -0,0 +1,111 @@
+description: Total CHIP spending in FY 2023 (millions of dollars)
```

Comment:
active voice and lets remove "FY2023" from all descriptions as we might add values in the future for other time periods